modules/core/src/main/java/io/fluo/webindex/core/models/Link.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2016 Fluo authors (see AUTHORS)
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.fluo.webindex.core.models;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+public class Link implements Serializable {
+
+  private static final long serialVersionUID = 1L;
+
+  private String url;
+  private String pageID;
+  private String anchorText;
+
+  public Link() {}
+
+  public Link(String url, String pageID, String anchorText) {
```

Comment:
small potential for bugs here... do not expect caller of method to pass in a url thats unerelated to page id or visa versa